### ISSUE TITLE
Restructure sidebar for improved mobile UX

### ISSUE THREAD

**Issue by Anjellyrika** [BastiDood/drap#164]
The mobile sidebar is currently toggled through a sidebar trigger button on the top left corner of the screen. For improved accessibility, the sidebar trigger should be moved to the bottom of the screen, ideally as part of a bottom nav component.

## Bottom navigation for mobile
- Should include links to Home, Dashboard and the sidebar trigger now shown as a hamburger menu icon
- The sidebar content itself should stay as is

## Dashboard redirects
- Must confirm for each user type that they are brought to the appropriate page when tapping the Dashboard button

## Prefer Drawer root component for mobile
- To allow closing the sidebar when tapping outside of it, replace the Sidebar root component with Drawer root which implements this behavior by default
- This should only apply for mobile; use a width media query to choose the correct root component

**Comment by Enoux** [comment]
sana machoose 🤞
